app/src/components/account/UserMenu.vue

```diff
@@ -49,22 +53,42 @@
           <div class="address-item">
             <img
               class="network-icon"
-              :src="address.icon"
+              :src="address.icon || `/img/networks/${address.networkId}.png`"
               alt="little circle with network logo"
             />
             <div>
-              <span class="address-network">{{
-                getAddressNetwork(address).title
-              }}</span>
+              <div class="address-name-role">
+                <p class="address-name">
+                  {{ getSessionName(address) || `Unknown` }}
```

Comment:
```suggestion
                  {{ getSessionName(address) }}
```
you have a default case so i don't think this is ever used

Reply:
Wait, there is no default. I didn't look at it, but yes, there is no default here